docs/batch-metrics-api.md

```diff
@@ -0,0 +1,160 @@
+# Batch Metrics API
+
+## Background and Problem statement
+The metrics API currently allows us to collect the past 5 seconds of performance data from an ODFE cluster. However, this data is aggregated, not raw, performance metrics. Additionally, it is restricting that we only have access to the past 5 seconds of data, rather than data from longer periods of time. Having fine granularity (raw) metrics from longer periods of time would allow users to learn a lot more about the performance of an AES cluster.
```

Comment:
nit: fine granularity doesn't mean raw, it means smaller rollup windows like 1s vs 5s vs 30s for example

Reply:
It's finer granularity if you measure by datapoints rather than time.